animesearch-ui/src/App.js

```diff
@@ -1,14 +1,49 @@
 import logo from './logo.svg';
 import './App.css';
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
 
 function App() {
+  const [searchData, setSearchData] = useState()
+
+  // useEffect(() => {
+  //   axios.get('https://kitsu.io/api/edge/anime?filter[text]=cowboy%20bebop')
+  //   .then(response => {
+  //     alert('searchData ' + JSON.stringify(response.data))
+  //     if (response.data.length > 0) {
+  //       setSearchData({...searchData, searchData: response.data})
+  //       alert('searchData ' + JSON.stringify(response.data))
+  //     }
+  //   })
+  //   .catch((error) => {
+  //     console.log(error);
+  //   })
+  // }) // this second arg is a dependency array
+
+  const handleSubmit = (e) => {
+    // removed appended &send_at=${email.sendAt} on fetch
+    alert('searchData ' + JSON.stringify(searchData));
+    axios.get(`https://kitsu.io/api/edge/anime?filter[text]=${JSON.stringify(searchData)}`)
```

Comment:
does this work without the token and key?
also i thought we have another card for integrating the backend with ui so ideally you commit your changes with regards to backend integration in that other card

Reply:
also forgot to mention, you should have the environment related info into a separate config.js; for example it should contain:
- backendApiUrl
- token
- key

As these would be different for dev (mock api for example) vs prod (actual api url)